signer/tests/integration/emily.rs

```diff
@@ -443,42 +497,12 @@ async fn deposit_e2e() {
         .unwrap()
         .is_empty());
 
-    // Add a stacks block in the current bitcoin tip, otherwise `get_last_key_rotation`
-    // will not get the signer keys and the txcoord process_new_blocks will fail.
-    // TODO: remove after #559 is fixed
-    assert!(context
-        .get_storage()
-        .get_last_key_rotation(&deposit_block_hash.into())
-        .await
-        .unwrap()
-        .is_none());
-
     let stacks_tip = context
         .get_storage()
         .get_stacks_chain_tip(&bitcoin_chain_tip.block_hash)
         .await
         .unwrap()
         .unwrap();
-    sqlx::query(
```

Comment:
Correct to assume that these hang together with the above comment about #559?

Reply:
Yes -- I tried to "enclose" the comment relevant block with the lone `//` below; we still use `stacks_tip` to check that the emily update has the correct information at the end of the test.